packages/@ama-sdk/create/README.md

```diff
@@ -17,6 +17,8 @@ npm create @ama-sdk typescript <package-name> -- [...options]
 
 ### Yarn
 
+This feature is experimental for yarn package manager
```

Comment:
not really aligned with the code that process.exit :D

Reply:
oups :) updated and show the message only if the run is failing